libdnf/base/transaction.cpp

```diff
@@ -700,6 +700,9 @@ Transaction::TransactionRunResult Transaction::Impl::_run(
                 system_state.set_group_state(group.get_groupid(), state);
                 // save the current xml group definition
                 group.serialize(comps_xml_dir / (group.get_groupid() + ".xml"));
```

Comment:
I am wondering if we want to keep the serialized group xml.
It would probably be cleaner to also delete it?

Reply:
Good point.

Reply:
I've added code for deleting the file.